client/src/plugins/user-journey-statistics/event-handlers/TabEventHandler.js

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
`TabEventHandler` sounds pretty generic. How many tab events do we expect, should we maybe split the event handlers?

Reply:
I don't expect new events. By tab events I meant opening and closing diagrams. We use `<diagram-type>.modeler.created`  events for `diagram:opened` and I added the `tab:close` event for `diagram:closed`, which can be used to establish the user journey.

Reply:
👍